test/distrib/bazel/python/namespaced/upper/example/BUILD

```diff
@@ -112,16 +112,16 @@ py_grpc_library(
 # in the following directory layout for python, which needs to properly be added to the imports.
 #
 # No Import or Strip (Can't compile if there are any proto dependencies)
-# bazel-out/darwin-fastbuild/bin/namespaced/upper/example/namespaced_example_pb2.py
```

Comment:
I'm not familiar with bazel/python. Do we need another reviewer or this should be ok?